pocs/camera/__init__.py

```diff
@@ -81,7 +81,7 @@ def kwargs_or_config(item, default=None):
     logger.debug("Camera config: {}".format(camera_info))
 
     simulator_names = hardware.get_simulator_names(config=config, kwargs=kwargs)
-    logger.debug(f'simulator_names = {", ".join(simulator_names)}')
+    logger.debug('simulator_names = {}'.format(", ".join(simulator_names)))
```

Comment:
Did you intend to revert this?

Reply:
Yes, it doesn't work with the `join` in the f-string and caused a bug. Really it should be it's own PR.

Reply:
Hm, odd. I'll try to recreate the problem on PAN001.

I think it was coming through as some mixed type, e.g.:
```
In [1]: lst = ['a', None]

In [2]: f'lst = {", ".join(lst)}'
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-296f01098e95> in <module>()
----> 1 f'lst = {", ".join(lst)}'

TypeError: sequence item 1: expected str instance, NoneType found
```

But that fails outside the f-string anyway.

Reply:
Oh, hm. PAN001 is on python 3.5.

Reply:
I'll remove from this PR.